src/main/java/core/basesyntax/service/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitRecord;
+import java.util.List;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private ParseService parseService;
+    private OperationHandlerService operationHandlerService;
+
+    public FruitShopServiceImpl() {
+        this.parseService = new ParseServiceImpl();
+        this.operationHandlerService = new OperationHandlerServiceImpl();
+    }
+
+    @Override
+    public void addInfoToStorage(List<String> rowsList) {
```

Comment:
In this method you parse lines to some model and carry out operations on the storage, this violates SRP, let's split this logic into 2 separate services

Reply:
fixed.
added RecordListMakerService and StorageInfoUpdateService